Formula/crosstool-ng.rb

```diff
@@ -23,12 +24,25 @@ class CrosstoolNg < Formula
   depends_on "grep"
   depends_on "libtool"
   depends_on "m4"
+  depends_on "make"
   depends_on "ncurses" if DevelopmentTools.clang_build_version >= 1000
   depends_on "xz"
 
+  if build.head?
+    depends_on "bash"
+    depends_on "bison"
+    depends_on "gettext"
+  end
+
   def install
+    if build.head?
+      system "./bootstrap"
+      ENV["BISON"] = "#{Formula["bison"].opt_bin}/bison"
+      ENV.append "LDFLAGS", "-lintl"
+    end
+
     ENV["M4"] = "#{Formula["m4"].opt_bin}/m4"
-    ENV["MAKE"] = "/usr/bin/make" # prevent hardcoding make path from superenv
+    ENV["MAKE"] = "#{Formula["make"].opt_bin}/gmake"
```

Comment:
When we set `MAKE` to system make it's going to be used as `make` for building toolchains in `crosstool-ng`. I added dependency on `gmake` because it's not possible to compile glibc 2.28 with [make < 4.0](https://sourceware.org/git/?p=glibc.git;a=blob;f=configure.ac;h=6cc10ede988ee5068e5f100a94beb95e48ee9edf;hb=HEAD#l995) and Apple ships 3.81.

Reply:
@fxcoudert is there a reason to depend on standard make?